test/manual/socks5.test.ts

```diff
@@ -17,6 +17,9 @@ describe('Socks5 Connectivity', function () {
     return;
   }
 
+  // 'TODO: NODE-3891 - fix tests broken when AUTH enabled'
+  const maybeIt = process.env.AUTH === 'auth' ? it.skip : it;
```

Comment:
These tests are worth taking a gander at why they're an issue, a socks connection will always be used in an auth scenario, I'm confident that the failure we're experiencing here are a miss-config as opposed to a real issue, any chance you have reference to an error message?

Reply:
I definitely was seeing failures before but now I don't, for some reason.  Maybe I fixed it somewhere else?  Regardless, I re-enabled them and ran a patch [here](https://spruce.mongodb.com/version/61facc97d6d80a4fb31284ad/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC) and they're passing so I reverted this change.